Task_8/index.html

```diff
@@ -1,99 +0,0 @@
-<!DOCTYPE html>
```

Comment:
Please revert out Task_8 folder or you our task_8 will have merge conflicts

Reply:
done